enterprise/frontend/src/metabase-enterprise/embedding-sdk/index.js

```diff
@@ -1,13 +1,6 @@
-import { SimpleDataPicker } from "embedding-sdk/components/private/SimpleDataPicker";
 import { PLUGIN_EMBEDDING_SDK } from "metabase/plugins";
 import { hasPremiumFeature } from "metabase-enterprise/settings";
 
 if (hasPremiumFeature("embedding_sdk")) {
   PLUGIN_EMBEDDING_SDK.isEnabled = () => true;
 }
-
-/**
- * We can't gate this component behind a feature flag, because SDK users could
- * use the SDK without a valid license and doesn't contain any feature flags.
- */
-PLUGIN_EMBEDDING_SDK.SimpleDataPicker = SimpleDataPicker;
```

Comment:
this is moved to a more appropriate place as this SimpleDataPicker isn't only for SDK anymore. It's also used in the interactive embedding.

Moving this to a seprate plugin doesn't change it's behavior because it isn't under any feature flag check, so we're safe.